blog/_posts/2021-11-12-ahed-project.md

```diff
@@ -0,0 +1,392 @@
+
+As a recent addition to the ADS Team supporting curation efforts and assisting in collection management, this appealed to my interests as I have recently begun honing my Python skills, and learning new tools such as Jupyter Notebook and OpenRefine. Jupyter Notebook is especially useful for new Python users because it helps break up scripts into more manageable blocks (cells) and I can include my notes, findings, and documentation along the way.
+ 
+In this documentation I intend to outline the goals I established, the steps I took to accomplish them, and lessons learned. To accomplish this project, I used a combination of my own knowledge and expertise, I read API Documentation, searched online for solutions as needed, and I also collaborated with team members to debug and learn the ins and outs of the ADS API.
+
+### Project Outline and Goals
+
+The source data used in this project was an excel spreadsheet provided of AHED’s bibliographic holdings (with metadata for Authors, Org code, Title, Journal information, and DOI if available), which was split up into three sheets by Branch (Astrophysics Branch, the Planetary Systems Branch, and the Exobiology Branch). 
+
+<b>Sample image of original excel file, showing items in three tabs for the AHED Branches</b>
+
+<img width="1004" alt="Screen Shot 2021-11-04 at 11 44 23 AM" src="https://user-images.githubusercontent.com/31739067/140366721-e2721366-94d7-4651-a06d-17e7a10483e4.png"> 
+
+The main overall goal was to match each publication with an ADS bibcode if it exists. In order to accomplish this, my team recommended I use the ADS API to match based on DOIs (for those provided), then reference strings, and then fill in the rest by Title. I split up my overall goal into four major tasks, and for each of these phases I created a new Jupyter Notebook and outlined the steps.
+1. [(Notebook 1): Match AHED to ADS Items by DOI](#match-by-doi)
+2. [(Notebook 2): Match AHED to ADS Items by Reference Strings](#match-by-ref)
+3. [(Notebook 3): Match AHED to ADS Items by Title](#match-by-title)
+4. [(Notebook 4): Curate missing items and create ADS Libraries](#ads-libs)
+
+## <a name="match-by-doi">Task 1 (Notebook 1): Match AHED to ADS Items by DOI</a>
+
+Create a version of the AHED spreadsheet that has a column "bibcode" added to the right of the DOI column. For those publications we are able to match to ADS records, this column will list these bibcodes, otherwise "NA".
+
+<b>Notebook 1 Outline:</b>
+<li>Step 1.1: Data Cleanup and Prep - combine all the excel sheets to one data set
+<li>Step 1.2: Isolate/create list of DOIs
+<li>Step 1.3: API Connection & Query
+<li>Step 1.4: Match list of bibcodes to original data set
+
+<b>Step 1.1: Data Cleanup and Prep</b>
```

Comment:
So, I'll leave this up to you, but if you want, we can shorten how the blog post looks on the page by collapsing some of the details into little closed sections that people can click to open (see any of the "development details" sections on the What's New page in the help docs to see what I mean). Pros: looks shorter at first glance, cons: people have to click more to read, which they may not be into. If you want to go for it, I'd probably advocate for just one closed section per task/notebook, that collapses everything after the outline into one sub-section. But again, up to you. We can play around with it together to see what it looks like if you're potentially interested.